index.js

```diff
@@ -1,16 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor(items,length) {
+    this.items=[];
+    this.length= this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.item=item;
+    this.items.push(item);
+    this.length +=1;
+    this.items.sort((a,b)=>{a-b});
```

Comment:
good logic, but the function called by `sort` should **return** the value (in this case `a - b`) for the function to work properly. Otherwise the array will not be sorted